1-exercises/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,20 @@
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+function printing (num) {
+  if (num === 3 || num === 6 || num === 9 || num === 12){
+    console.log("Fizz");}
+    else if (num === 5 || num === 10) {
+      console.log("Buzz");}
+      else if (num === 15) {
+        console.log("FizzBuzz")
+      } else {
+        console.log(num);
+      }
+    }
+
+arr.forEach(printing);
```

Comment:
your solution works for the array given, well done :) can you think of another way of approaching this which uses the remainder operator? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder
For example, what would happen with your solution above if the array contained numbers beyond 15?